library/network/src/lib/y2firewall/firewalld/service.rb

```diff
@@ -86,6 +89,8 @@ def self.modify_ports(name:, tcp_ports: [], udp_ports: [])
       # @param name [String] zone name
       def initialize(name:)
         @name = name
+
+        @modified = []
```

Comment:
um, is it ever used for service?

Reply:
Yes as service is also using relations.

Reply:
yes. I got into that metaprogramming stuff a bit an see now.

Reply:
just a note, 
shouldn't  the stuff around "modified" be part of that metaprogramming block? It is used mainly there anyway.